### PR TITLE
feature (refs T31096): Show statement attachments in originalSTNView

### PR DESCRIPTION
**Pull request by MoritzMandler**
**Ticket:** https://yaits.demos-deutschland.de/T31096

**Description:**

- in AT in the tab `Originalstellungnahme` you couldnt see the `Originalstellungnahme-Anhang`, only the `weitere Anhänge`
- in FE in store the relationship were in Singular, it should be in Plural because in the rules of JSON.API it requires this way

stops filtering attachments of original statements by skipping
the condition within the ```getAccessCondition``` of
the ```StatementResourceType``` when called from
the ```StatementAttachmentResourceType```.
Activate the needed permission to include
STN-attachments via api.

- now you can see the original attachment in the AT in the first assessment table view, too, before switching the tab to the original statement

**How to review:**

- go to AT, click to `Originalstellungnahmen`, you will see here `OriginalSTN-Anhang` as attachment
- if you dont have any attachment here, please insert one

### Linked PRs 
Robobsh Repo: https://github.com/demos-europe/demosplan-project-robobsh/pull/20

- [x] Tests updated/created
- [x] Link all relevant tickets
- [x] Move the tickets on the board accordingly

### CODE REVIEW

**Review by hwiem — APPROVED** [review]
FE lgtm, code review only

File: demosplan/DemosPlanCoreBundle/ResourceTypes/StatementResourceType.php [hunk 1]
```diff
@@ -140,21 +140,25 @@ public function buildAccessCondition(StatementResourceType $pathStartResourceTyp
...
-        );
+        ];
+        if (!$allowOriginals) {
+            // Normally the path to the relationship would suffice for a NULL check, but the ES
+            // provides the 'original.id' path only hence we need the path to the ID to support
+            // ES queries beside Doctrine.
+            $conditions[] = $this->conditionFactory->propertyIsNotNull($pathStartResourceType->original->id);
+        }
+
+        return $this->conditionFactory->allConditionsApply(...$conditions);
```
[the remark this answers is not in the document]
Comment (MoritzMandler) [reply demosplan/DemosPlanCoreBundle/ResourceTypes/StatementResourceType.php:161]: done in d0dd2db44aea8d2e76f589e5aadbaebee5e8c1a5

**Review by spiess-demos — APPROVED** [review]
fe lgtm

**Review by martendemos — APPROVED** [review]
lgtm: code review and local test

**Pull request merged**